article/sections/_3_graph_overview.tex

```diff
@@ -0,0 +1,160 @@
+\section{Overview of Causal Graphs}
+\label{sec:graph-overview}
+The previous section, \ref{sec:graph-importance}, motivated why the knowledge encoded in one's DAG is important. 
+This section gives a high level overview of what DAGs are and provides references for more thorough readings on the subject. 
+Causal diagrams and causal graphical models have been introduced by \citet{pearl_causality_1995} as a powerful tool for causal inference, especially in observational studies.
+Perhaps one of the most important and useful features of causal
+graphs when dealing with causal inference problems is the clear illustration
+of the causal relationships between the variables.
+While a formal introduction to the topic of DAGs is beyond the scope of this chapter,
+here we focus specifically on illustrating the power of DAGs to represent
+and encode complex causal relationships between variables in an intuitive and
+clear manner.
+Interested readers can refer to \citet{pearl_causality_2000} for a thorough introduction.
+
+Consider the causal graph represented in Figure \ref{fig:simple-graph}.
+Suppose we're interested in the effect of a treatment Z on Y.
+What the graph in Figure \ref{fig:simple-graph} implies is that Z is
```

Comment:
I think it'd be good to explain why the graph implies this (i.e. something similar to the fact that there is no path from Y to Z without going through X).

Reply:
See edits in #176.